registries.conf

```diff
@@ -8,6 +8,11 @@
 #
 # Registries to search for images that are not fully-qualified.
 # i.e. foobar.com/my_image:latest vs my_image:latest
+# NOTE: pulling an image that is not fully qualified, i.e. one that
```

Comment:
I think this needs more description.  Let's use a google doc to start discussion.